ansible/roles/intellij/tasks/main.yml

```diff
@@ -91,3 +91,10 @@
     file:
       path=/opt/intellij/IC/intellij.sha256
       state=absent
+
+  - name: ensure java 8 is set as default
+    alternatives:
+      link: /usr/bin/java
+      name: java
+      path: /usr/lib/jvm/java-8-openjdk-amd64/bin/java
+      priority: 100
```

Comment:
I'm happy to move that into a separate `java` role but I thought it's IntelliJ specific and it's easier to understand why this exists if it's under `intellij`

Reply:
Nah that's cool. We can remove it once we solve the case of the rogue java installation